lib/rbs/prototype/rb.rb

```diff
@@ -393,7 +405,7 @@ def literal_to_type(node)
             value_types << literal_to_type(v)
           end
 
-          if key_types.all? { |t| t.is_a?(Types::Literal) }
+          if !key_types.empty? && key_types.all? { |t| t.is_a?(Types::Literal) }
```

Comment:
An empty record is syntax error, so it should be `Hash[untyped, untyped]` instead.